ansible/inventories/development/hosts

```diff
@@ -1,5 +1,5 @@
 [manager]
-manager.development.inaf.it ansible_host=192.168.10.200
+discosdev.inaf.ira.it ansible_host=192.168.60.43
```

Comment:
This seems to me like a custom configuration. Maybe it was pushed by mistake? I'd rather keep the old address for the virtual machine, this one seems related to IRA only...

Reply:
Sorry, it was pushed by mistake.